compiler/modules/types/list.bal

```diff
@@ -236,6 +236,29 @@ function listFormulaIsEmpty(Context cx, Conjunction? pos, Conjunction? neg) retu
     return !listInhabited(cx, indices, memberTypes, nRequired, neg);
 }
 
+function intersectListAtoms(Env env, ListAtomicType[] atoms) returns [SemType, ListAtomicType]? {
+    if atoms.length() == 0 {
+        return ();
+    }
+    ListAtomicType atom = atoms[0];
+    foreach int i in 1 ..< atoms.length() {
+        ListAtomicType next = atoms[i];
+        var intersection = listIntersectWith(atom.members, atom.rest, next.members, next.rest);
+        if intersection is () {
```

Comment:
Don't like `intersection` as a name (`atom` is also the intersection)

Reply:
Fixed. Changed to `tmpAtom` to match `mapping.bal`

Reply:
Fixed. Changed to `tmpAtom` to match `mapping.bal`